central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -589,7 +594,7 @@ func (s *flowStoreImpl) RemoveFlow(ctx context.Context, props *storage.NetworkFl
 }
 
 // RemoveMatchingFlows removes the specified flows from the store
-// Todo: Figure out what to do with the functions.
+// Todo: ROX-9921 Figure out what to do with the functions.
```

Comment:
Can you describe the logical relation for the flows to remove with function keyMatchFn and valueMatchFn? It is not explicit here. Like "removes the specific flows matching both keyMatchFn and valueMatchFn ..."